include/grpc++/completion_queue.h

```diff
@@ -36,6 +36,7 @@
 
 #include <chrono>
 #include <grpc++/impl/client_unary_call.h>
+#include <grpc/support/time.h>
```

Comment:
This was actually part of Craig's original objection to my using timespec - we have been pretty consistent about not using grpc/ headers inside grpc++ header files. This one is easy enough to fix if the Next code is moved out-of-line.